server/routes/server.js

```diff
@@ -11,8 +11,20 @@ function convertToClientFormat(selected_config, esResponse) {
       var flatten = require('flat');
       source = flatten(source);
     }
+    
+    var display = source[selected_config.fields.mapping['display_timestamp']];
+    
+    if (selected_config.format_timestamp != null){
+      var moment = require('moment-timezone');
+      display = moment(source[selected_config.fields.mapping['timestamp']]);
```

Comment:
I meant shall we replace `display moment(source[selected_config.fields.mapping['timestamp']]);` with
`display = moment(source[selected_config.fields.mapping['display_timestamp']]);`? So that user has the flexibility to use a different timestamp field for display with custom format using `format_timestamp`